pipelines/library/src/common/RepoHandler.groovy

```diff
@@ -18,51 +18,51 @@ class RepoHandler {
         this.context = context
         this.configs = configs
 
-        def getAdopt = new URL(ADOPT_DEFAULTS_FILE_URL).openConnection()
+        def getAdopt = new URL(ADOPT_JENKINS_DEFAULTS_URL).openConnection()
         this.ADOPT_DEFAULTS_JSON = new JsonSlurper().parseText(getAdopt.getInputStream().getText()) as Map
     }
 
     /*
-    Getter to retrieve user's git remote config
+    Getter to retrieve user's git ci-jenkins-pipeline remote config
     */
     public Map<String, ?> getUserRemoteConfigs() {
         return configs
     }
 
     /*
-    Getter to retrieve adopt's defaults
+    Getter to retrieve adopt's ci-jenkins-pipelines defaults
     */
     public Map<String, ?> getAdoptDefaultsJson() {
         return ADOPT_DEFAULTS_JSON
     }
 
     /*
-    Getter to retrieve user's defaults
+    Getter to retrieve user's ci-jenkins-pipelines defaults
     */
     public Map<String, ?> getUserDefaultsJson() {
         return USER_DEFAULTS_JSON
     }
 
     /*
-    Setter to retrieve and save a user defaults json inside the object
+    Setter to retrieve and save a user ci-jenkins-pipelines defaults json inside the object
     */
     public Map<String, ?> setUserDefaultsJson(String url) {
         def getUser = new URL(url).openConnection()
         this.USER_DEFAULTS_JSON = new JsonSlurper().parseText(getUser.getInputStream().getText()) as Map
     }
 
     /*
-    Changes dir to Adopt's repo
+    Changes dir to adopt's ci-jenkins-pipelines repo
     */
     public void checkoutAdopt () {
```

Comment:
I think these 3 functions could be named better? say:
checkoutAdoptPipelines()
checkoutUserPipelines()
checkoutAdoptBuild()